client/components/accordion/index.jsx

```diff
@@ -14,6 +14,7 @@ import AccordionStatus from './status';
 export default class Accordion extends Component {
 	static propTypes = {
 		initialExpanded: PropTypes.bool,
+		temporaryExpand: PropTypes.bool,
```

Comment:
Should be an adverb — `temporarilyExpand` — though might I suggest something like `forceExpand`?